erpnext/crm/doctype/lead/lead_dashboard.py

```diff
@@ -5,8 +5,8 @@ def get_data():
 	return {
 		'fieldname': 'lead',
 		'non_standard_fieldnames': {
-			'Quotation': 'party_name',
-			'Opportunity': 'party_name'
+			'Quotation': 'customer_name',
+			'Opportunity': 'customer_name'
```

Comment:
Tested?

Not working for with customer_name.

- Working with a filter value party_name
<img width="430" alt="Screen Shot 2019-05-13 at 12 33 40 PM" src="https://user-images.githubusercontent.com/3784093/57602239-3e1d1a80-757c-11e9-92e6-2932f2dc33fb.png">
<img width="704" alt="Screen Shot 2019-05-13 at 12 36 04 PM" src="https://user-images.githubusercontent.com/3784093/57602240-3e1d1a80-757c-11e9-9e13-cac31516f672.png">

- Not Working with a filter value customer_name
<img width="547" alt="Screen Shot 2019-05-13 at 12 36 21 PM" src="https://user-images.githubusercontent.com/3784093/57602257-49704600-757c-11e9-8e81-666f57049a0a.png">
<img width="739" alt="Screen Shot 2019-05-13 at 12 36 27 PM" src="https://user-images.githubusercontent.com/3784093/57602258-4a08dc80-757c-11e9-9d3e-4ecacc4e86fc.png">